styles/all/template/js/collapsiblecategories.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {  // Avoid conflicts with other libraries
+
+	'use strict';
+
+	$(function () {
+		$('a.category[data-id] + .forabg').each(function () {
+			var $this = $(this),
+				id = $this.prev().attr('data-id'),
+				hidden = $this.prev().attr('data-hidden'),
+				$header = $this.find('li.header'),
+				$content = $this.find('.topiclist.forums');
+
+			if (!$header.length || !$content.length) {
+				return;
+			}
+
+			// Add button
+			$header.append('<a class="collapse-btn collapse-' + ((hidden) ? 'show' : 'hide') + '" data-id="' + id + '" data-ajax="phpbb_collapse"></a>');
+
+			// Hide hidden forums on load
+			if (hidden) {
+				$content.hide();
+			}
+		});
+
+		// Since we dynamically create our AJAX button, phpBB's does not ajaxify it
+		// so we reproduce phpBB's ajaxify on our buttons here
+		$('.collapse-btn').each(function () {
```

Comment:
Very interesting... Because you put the INCLUDEJS in overall_header, our script gets loaded before phpBB's ajax.js. Which means we do not need this function (lines 26~40). Right now our button is being double-ajaxified, so we should remove this code :)